src/main/webapp/js/instructorStudentRecords.es6

```diff
@@ -102,23 +103,23 @@ function submitCommentForm(commentIdx) {
  * Blanks are not allowed.
  */
 function checkComment(form) {
-    var formTextField;
+    let formTextField;
 
-    if ($(form).find('[id^=' + COMMENT_EDITTYPE + ']').val() === 'delete') {
-        return true;
+    if ($(form).find(`[id^=${COMMENT_EDITTYPE}]`).val() === 'delete') {
+        return;
     }
 
     if ($(form).attr('name') === 'form_commentadd') {
         formTextField = tinymce.get(COMMENT_TEXT).getContent();
     } else {
-        var editorId = $(form).attr('id').match(/^form_commentedit-(\d+)$/)[1];
-        formTextField = tinymce.get('commentText' + editorId).getContent();
+        const editorId = $(form).attr('id').match(/^form_commentedit-(\d+)$/)[1];
+        formTextField = tinymce.get(`commentText${editorId}`).getContent();
     }
 
     if (isBlank(formTextField)) {
         setStatusMessage(DISPLAY_COMMENT_BLANK, StatusType.DANGER);
         scrollToTop();
-        return false;
+        form.preventDefault();
```

Comment:
Wait a minute, `form` is not an event object. In fact, there is no event object involved here. (so this `return false;` should have been doing nothing all these while?)